ngx-tools/type-guards/src/is-mouse-event/is-mouse-event.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { createMouseEvent } from '@terminus/ngx-tools/testing';
+
+import { isMouseEvent } from './is-mouse-event';
+
+
+describe(`isMouseEvent`, function() {
+
+  test(`should return true for mouse events`, function() {
+    const fakeMouseEvent = createMouseEvent('click');
```

Comment:
other good mouse events: doubleclick, drag, drop, mouseup, mousedown, mouseenter, etc

Reply:
Adding these would just test our `createMouseEvent` function since nothing in `isMouseEvent` is looking at things like keycodes or the event name etc